src/inc/quic_platform_winuser.h

```diff
@@ -671,8 +671,17 @@ typedef HANDLE CXPLAT_EVENT;
 #define CxPlatEventSet(Event) SetEvent(Event)
 #define CxPlatEventReset(Event) ResetEvent(Event)
 #define CxPlatEventWaitForever(Event) WaitForSingleObject(Event, INFINITE)
-#define CxPlatEventWaitWithTimeout(Event, timeoutMs) \
-    (WAIT_OBJECT_0 == WaitForSingleObject(Event, timeoutMs))
+inline
+BOOLEAN
+_CxPlatEventWaitWithTimeout(
+    _In_ CXPLAT_EVENT Event,
+    _In_ uint32_t TimeoutMs
+    )
+{
+    CXPLAT_DBG_ASSERT(TimeoutMs != UINT32_MAX);
+    return WAIT_OBJECT_0 == WaitForSingleObject(Event, TimeoutMs);
+}
+#define CxPlatEventWaitWithTimeout(Event, TimeoutMs) _CxPlatEventWaitWithTimeout(Event, TimeoutMs)
```

Comment:
Why have this macro at all? Just use an inline function?

Reply:
Was following the CxPlatEvent patterns in quic_platform_winkernel.h and quic_platform_winuser.h?

Reply:
Ok, will update. Also it appears we are hitting this new assert in functional tests for both winkernel and winuser platforms. I might end up reverting the assert addition (and maybe open an issue to investigate this incorrect API usage?)